File: doc/src/scripts/ServerStats.py
import krpc
conn = krpc.connect()
status = conn.krpc.get_status()
print('Data in = %.1f KB/s' % (status.bytes_read_rate/1024.0))
print('Data out = %.1f KB/s' % (status.bytes_written_rate/1024.0))

File: doc/src/scripts/ServerStats.lua
local krpc = require 'krpc'
local conn = krpc.connect()
local status = conn.krpc:get_status()
print(string.format('Data in = %.2f KB/s', status.bytes_read_rate/1024))
print(string.format('Data out = %.2f KB/s', status.bytes_written_rate/1024))
